dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/threadutils/ThreadUtilsTest.java

```diff
@@ -77,13 +77,12 @@ public void run() {
                     start.add(Calendar.SECOND,  1);
                     globalTimer.add(Calendar.SECOND, 1);
                 }
-                System.out.println("time is " + System.currentTimeMillis());
             }
         };
         scheduleService.scheduleAtFixedRate(schedulerTask, 2, 10, TimeUnit.SECONDS);
         assertFalse(scheduleService.isShutdown());
         try {
-            Thread.sleep(60000);
+            Thread.sleep(2000);
```

Comment:
Sleep is not recommended in the test code, maybe we should try to use Awaitility to achieve the same effect?